packages/cms-lib/hubdb.js

```diff
@@ -147,7 +165,16 @@ async function downloadHubDbTable(portalId, tableId, dest) {
   await fs.writeFileSync(dest, tableJson);
 }
 
+async function clearHubDbTable(portalId, tableId) {
+  const rows = await fetchRows(portalId, tableId);
```

Comment:
This code only clears the first page of rows not all rows since fetching won't automatically fetch all rows.

Reply:
Good point, I've updated this